.github/scripts/setup_runners_service.sh

```diff
@@ -0,0 +1,187 @@
+#!/bin/bash
+
+# This script allows users to manually assign parameters
+if [ "$#" -ne 2 ] || [ "$1" = "--help" ] || [ "$1" = "-h" ]
+then
+  echo "Please assign a2 parameters when running this script"
+  echo "Example: $0 \$SETUP_DISTRO \$SETUP_ACTION"
+  echo "Example: $0 \"tar | deb | rpm\" \"--es | --es-nosec | --kibana | --kibana-nosec\""
+  exit 1
+fi
```

Comment:
Describe parameters

Reply:
Now sure what that means, as I already have 2 examples showing what users need to enter.